.travis/install_clang_tools.sh

```diff
@@ -4,7 +4,6 @@ set -ex
 output_dir=./build
 cmake_version=3.12.0
 # install cppcheck
-sudo apt-get update
```

Comment:
this is not a good way to resolve this, maybe better to rebase my diff to solve the problem